perfkitbenchmarker/providers/rackspace/flags.py

```diff
@@ -14,37 +14,39 @@
 
 from perfkitbenchmarker import flags
 
-flags.DEFINE_boolean('boot_from_cbs_volume', 'False',
-                     'When flag is included the instance will use a remote disk'
-                     'as its boot disk, if machine_type supports it.')
+flags.DEFINE_string('rack_path',
+                    default='rack',
+                    help='The path for the rack CLI binary.')
 
-flags.DEFINE_string('nova_path',
-                    'nova',
-                    'The path for the rackspace-novaclient tool.')
+flags.DEFINE_string('region', default='IAD',
+                    help='A string indicating which Rackspace region to use.')
 
-flags.DEFINE_string('neutron_path',
-                    'neutron',
-                    'The path for the rackspace-neutronclient tool.')
+flags.DEFINE_string('profile', default=None,
+                    help='A string indicating which RackCLI profile to use. '
+                         'If none is specified default profile is used '
+                         '(see https://developer.rackspace.com/docs/'
+                         'rack-cli/configuration/#config-file)')
 
-flags.DEFINE_list('additional_rackspace_flags',
-                  [],
-                  'Additional flags to pass to Rackspace.')
+flags.DEFINE_boolean('boot_from_cbs_volume', 'False',
+                     'When flag is included the instance will use a remote disk'
+                     ' as its boot disk, if machine_type supports it'
+                     ' (Currently Standard and OnMetal flavors do not support'
```

Comment:
I am mildly worried about PKB's help text saying that Standard and OnMetal don't support boot-from-CBS, because then if you enable boot-from-CBS on Standard or OnMetal, we have to update PKB. It's just a bit more maintenance burden. However, up to you. :)